homeassistant/components/google_generative_ai_conversation/tts.py

```diff
@@ -0,0 +1,232 @@
+"""Text to speech support for Google Generative AI Conversation."""
```

Comment:
I'd drop the Conversation from all these comments. Including it in the domain was a mistake.

Reply:
Yeah, I got confused about the intent/scope of this integration because of it. Dropped it